news/_posts/2019-07-04-1.6.1.x-what-s-next.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: post
+title: "1.6.1.x: what’s next"
+subtitle: "The end of 1.6 official support by the PrestaShop company. And the beginning of something new."
+date:   2019-07-04 11:30:00
+authors: [ AntoineThomas ]
+icon: icon-flag-corner
+tags:
+ - 1.6
+ - Maintenance
+---
+
+
+This last year, the PS company has been facing a dilemma: how to dedicate 100% of the developers’ time to move forward faster on the 1.7 version of the open source project, something everybody agrees on. And, at the same time, provide more time for merchants still using the 1.6 version to get ready to upgrade, by keeping them safe in case of critical issue [after the official support ends on the 30th of June 2019[(https://www.prestashop.com/en/blog/maintenance-extension-prestashop-1-6). We have come up with a solution, thanks to the initiative and support of a few volunteers from the PrestaShop developer community: allowing them to take over the responsibility of maintaining PrestaShop 1.6.1.x beyond the end of official support.
+
+
+## What does it mean?
+
+Put it simply, from the 1st of July 2019, the maintenance of PrestaShop 1.6.1.x is now being performed exclusively by volunteers. If a critical issue is discovered (we think mainly of security vulnerabilities), this team of volunteers will look for solutions from the community, work on it through a process of code review and extensive testing, in order to release a patch version to solve this critical issue.
+
+Those patch versions will not be officially issued by PrestaShop nor available on prestashop.com, instead, they will be available on a dedicated GitHub repository:
+[https://github.com/PrestaShop/PrestaShop-1.6](https://github.com/PrestaShop/PrestaShop-1.6)
+
+On this repository, it will be possible to submit pull requests in order to fix critical issues to be shipped in a patch version for PrestaShop 1.6.1.x.
+
+
+## Scope of the maintenance
+
+The volunteers and the PrestaShop company have agreed on the following objectives:
+
+Fix critical issues (like the recent spam attack on account creation form) or security issues
+Eventually, if this is considered possible, allow PrestaShop 1.6 to be run with PHP 7.2
+
+Consequently, non critical issues and pull requests for the 1.6 version which are not related to one of those objectives will not be merged and will be archived. This will help the new maintainers stay focused on critical patch versions. 
+
+If you find a security vulnerability in the software, the process stays the same:
+
+- Don’t speak about it publicly immediately
+- Instead, please send an email to security@prestashop.com with details about the issue, how to reproduce it, and any other interesting details
+- You can of course submit a patch proposal in the email
+- When the patch is approved and the release available, you can speak about it
+
+If you want to participate in making the 1.6.1.x work with PHP 7.2:
```

Comment:
Sometimes it's 1.6, sometimes 1.6.1.x :thinking:

Reply:
most of the time, it is 1.6.1.x. 
there too times where we speak about 1.7 and 1.6. But when this is about the work to come, this is always 1.6.1.x